packages/frontend/src/admin-pages/components/auth/forgotPassword/ForgotPasswordForm.vue

```diff
@@ -141,6 +141,7 @@ export default {
           vue.$router.push({ name: 'forgot-password-request-sent' })
         },
         onFailure: function (err) {
+          console.log(err)
```

Comment:
was this missed in cleanup?

Reply:
It was intentional. See https://eslint.org/docs/latest/rules/handle-callback-err